loginas/views.py

```diff
@@ -9,18 +12,7 @@
 from . import settings as la_settings
 from .utils import login_as, restore_original_login
 
-try:
```

Comment:
Hmm, the tests are fine, but I'd rather not remove this backwards-compatibility code for no reason, even if it's ancient.

Reply:
Ok, I dropped those commits, though I don't think that any code exists that it will make a difference for. It's still passing on my end.